events/lastAvatarStandingTrivia/entityServerScripts/triviaMasterServer.js

```diff
@@ -561,161 +566,163 @@
         }); 
     };
 
-    this.loseCoins = function(){        
-        Entities.addEntity({
-            type: "ParticleEffect",
-            name: "Trivia Particle Coin Lose",            
-            lifetime: 4,
-            collidesWith: "",
-            collisionMask: 0,
-            collisionless: true,
-            position:  coin.position,
-            dimensions: {
-                x: 0.15,
-                y: 0.15,
-                z: 0.15
-            },
-            isEmitting: true,                           
-            userData: "{\"grabbableKey\":{\"grabbable\":false}}",          
-            lifespan: 2,
-            maxParticles: 200,
-            textures: Script.resolvePath("../entities/pictures/coin.png"),
-            emitRate: 5,
-            emitSpeed: 0,
-            speedSpread: 5,
-            emitDimensions: {
-                x: 0,
-                y: 0,
-                z: 0
-            },
-            emitOrientation: {
-                x: 0,
-                y: 0,
-                z: 0,
-                w: 1
-            },
-            emitterShouldTrail: true,
-            particleRadius: 1,
-            radiusSpread: 0,
-            radiusStart: 1.25,
-            radiusFinish: 0,
-            color:{
-                red:255,
-                blue:0,
-                green:0
-            },
-            colorSpread:{
-                red: 0,
-                blue: 255,
-                green: 0
-            },
-            colorStart:{
-                red:255,
-                blue:0,
-                green:0
-            },
-            colorFinish:{
-                red: 0,
-                blue: 0,
-                green: 0
-            },
-            emitAcceleration:{
-                x:0,
-                y:-2,
-                z:0
-            },
-            accelerationSpread:{
-                x:2,
-                y:0,
-                z:2
-            },
-            alpha: 1,
-            alphaSpread: 0,
-            alphaStart: 1,
-            alphaFinish: 0,
-            particleSpin: 5,
-            spinSpread: 0,
-            spinStart: 0,
-            spinFinish: 0,
-            rotateWithEntity: true
-        });
-    };
+    // this.loseCoins = function(){  
+    //     return;      
```

Comment:
remove commented out code. If we need this code, it will be available in the old commit.

Reply:
This is still needed for trivia in the trivia domain, it was only removed for multi-con

Reply:
Remove